packages/fxa-content-server/app/scripts/lib/email-domain-validator.js

```diff
@@ -120,6 +120,9 @@ const checkEmailDomain = ($element, view) => {
           return reject(REJECTION_CODE);
         }
 
+        if(result==='skip') {
+          return resolve();
```

Comment:
resolved while making a skip record

Reply:
Can you put spaces around `===`?  I'm surprised that prettier didn't auto-format it.

Reply:
Instead of me nitpicking about code formatting, you could run prettier with `yarn prettier --write --config ../../_dev/.prettierrc FILE_NAMES`, e.g. `yarn prettier --write --config ../../_dev/.prettierrc app/scripts/lib/email-domain-validator.js`, from the fxa-content-server directory.

If you use VS Code, there's a prettier extension.

Reply:
No i am using prettier ext

Reply:
i made prettier to write this file now. so no issue regarding the linitng errors

Reply:
hmm are you using the prettierrc from the project?  I see different formatting when I run the command in my above comment?

Reply:
yup.